pkg/apis/flow/v1alpha1/xmltojsontransformation_lifecycle.go

```diff
@@ -29,7 +29,18 @@ const (
 	EventTypeXMLToJSONGenericResponse = "io.triggermesh.xmltojsontransformation.error"
 )
 
-var xmlToJSONCondSet = apis.NewLivingConditionSet()
+const (
+	// ConditionReady has status True when the router is ready to send events.
+	ConditionReady = apis.ConditionReady
+	// ConditionSinkProvided has status True when the router has been configured with a sink target.
+	ConditionSinkProvided apis.ConditionType = "SinkProvided"
+)
+
+var XMLToJSONCondSet = apis.NewLivingConditionSet(
+	ConditionReady,
```

Comment:
Unnecessary, `Ready` is already the default condition embedded in any Knative `ConditionSet`.

Example from another API group (notice that we don't explicitly declare the "Ready" condition):
https://github.com/triggermesh/triggermesh/blob/a7b8255c31533c309b24bb1e5f56222f1add9c38/pkg/apis/sources/v1alpha1/common_lifecycle.go#L64-L67